batchflow/models/tf/__init__.py

```diff
@@ -12,7 +12,7 @@
 from .inception_v4 import Inception_v4
 from .inception_resnet_v2 import InceptionResNet_v2
 from .squeezenet import SqueezeNet
-from .mobilenet import MobileNet, MobileNet_v2
+from .mobilenet import MobileNet_v1, MobileNet, MobileNet_v3, MobileNet_v3_small
```

Comment:
That's quite confusing. The original MobileNet is named MobileNet_v1, while the model named MobileNet is actually MobileNet v2.

Reply:
renamed.